src/Libraries/OrchardCore.Commerce.Abstractions/Exceptions/FrontendException.cs

```diff
@@ -1,11 +1,13 @@
 using Microsoft.AspNetCore.Mvc.Localization;
 using System;
-using System.Runtime.Serialization;
 
 namespace OrchardCore.Commerce.Abstractions.Exceptions;
 
 [Serializable]
+#pragma warning disable S3925 // "ISerializable" should be implemented correctly
+// Exception(SerializationInfo info, StreamingContext context) is obsolete and should not be called.
```

Comment:
Always put these comments above the pragmas.

Reply:
Also can you use `[SuppressMessage]` instead? I suggest only using `#pragma` inside method bodies. It should be avoided on class/property declarations and method signatures, because it's horrible for readability.